client/verta/verta/_cli/registry/update.py

```diff
@@ -19,7 +19,14 @@ def update():
 def update_model(model_name, label, workspace):
     """Create a new registeredmodel entry.
     """
-    pass
+    client = Client()
+    try:
+        registered_model = client.get_registered_model(model_name, workspace=workspace)
+    except ValueError:
+        raise click.BadParameter("model {} not found".format(model_name))
+    registered_model._msg.labels[:] = label[:]
```

Comment:
As per our thread in #model-registry, let's use `verta registry update` to _add_ labels instead of replacing them, for now.
```suggestion
    registered_model._msg.labels.extend(label)
```